cleanlab/datalab/internal/issue_manager/regression/label.py

```diff
@@ -58,50 +64,53 @@ def __init__(
         self,
         datalab: Datalab,
         clean_learning_kwargs: Optional[Dict[str, Any]] = None,
+        threshold: float = 0.1,
```

Comment:
```suggestion
        threshold: float = 0.13,
```

Reply:
consider this alternate default value that would match near duplicates

Reply:
disregard if that's a lot more work, or 0.1 is better default

Reply:
@elisno did you try 0.13 instead of 0.1?

Reply:
nevermind I see the false-positive rate is too high in the unit-test already, so let's not increase threshold